packages/kmod-5.15-nvidia/Cargo.toml

```diff
@@ -22,6 +22,16 @@ url = "https://us.download.nvidia.com/tesla/535.161.07/NVIDIA-Linux-aarch64-535.
 sha512 = "bb96a28b45197003480ae223c71a5426ef5258a31eaa485cab0cf4b86bed166482734784f20c6370a1155f3ff991652cac15f1b1083d2fb056677e6881b219e2"
```

Comment:
Commit message can just be: "add Fabric Manager support" since the "Bottlerocket" part is obvious in context.

Reply:
changed.